services/apps/alcs/src/decision/application-decision-meeting/application-decision-meeting.service.ts

```diff
@@ -72,6 +72,7 @@ export class ApplicationDecisionMeetingService {
       .innerJoin('meeting.application', 'application')
       .innerJoin('application.reconsiderations', 'reconsideration')
       .innerJoin('reconsideration.card', 'card')
+      .where("card.status_code != 'RELE'")
```

Comment:
Is it worth moving 'card.status_code != 'RELE' ' into constant and use at both places?